source/plugins/userprefrences.ts

```diff
@@ -0,0 +1,31 @@
+import debug from "../lib/debug";
+import {
+	Requirements,
+	requires,
+	type IPlugin,
+	type PluginInput,
+} from "../lib/pluginTypes";
+
+class UserprefrencePlugin implements IPlugin {
+	name = "Darkmode";
+	version = "0.0.1";
```

Comment:
```suggestion
	version = "1.0.0";
```
This is what the other main plugins use.